testcases/crypto/abfunc.c

```diff
@@ -565,24 +565,6 @@ void do_TestInvalidKeys(CK_SESSION_HANDLE session)
     CK_OBJECT_HANDLE pubkey = CK_INVALID_HANDLE, privkey = CK_INVALID_HANDLE;
     CK_RV rc;
 
-    // Expected to fail with CKR_TEMPLATE_INCOMPLETE
```

Comment:
So this removal is not related to the bug, right? 
Do I read it correctly that Dilithium keys are supported as AB keys?

Reply:
Ups, this line also makes sense for the remaining test cases.  Putting it back in.  But yes, Dilithium keys are actually supported for AB keys (undocumented feature).  So if Dilithium can be used, this test case will actually fail.